### PR TITLE
Some theorems for W-spaces

### PR DESCRIPTION
**Pull request by Almanzoris**
Perhaps, I should write something further in the description of the theorems? Like the statement.

### CODE REVIEW

File: theorems/T000580.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - P000187: true
+  - P000057: true
+then:
+  P000028: true
+refs:
+  - doi: 10.1016/0016-660X(76)90024-6
+    name: Infinite games and generalizations of first-countable spaces (Gruenhage)
+---
+
+Shown in the Theorem 3.3 of {{doi:10.1016/0016-660X(76)90024-6}}.
```
Comment (StevenClontz) [inline theorems/T000580.md:14]:
```suggestion
Shown in Corollary 3.4 of {{doi:10.1016/0016-660X(76)90024-6}}.
```

Shown a bit more directly here

File: theorems/T000581.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - P000011: true
+  - P000026: true
+then:
+  P000028: true
+refs:
+  - doi: 10.1016/0016-660X(76)90024-6
+    name: Infinite games and generalizations of first-countable spaces (Gruenhage)
+---
+
+Shown in the Theorem 3.6 of {{doi:10.1016/0016-660X(76)90024-6}}.
```
Comment (StevenClontz) [inline theorems/T000581.md:15]:
```suggestion
Shown in Theorem 3.6 of {{doi:10.1016/0016-660X(76)90024-6}}.
```

Grammar

File: theorems/T000582.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - P000003: true
+  - P000197: true
+then:
+  P000191: true
+refs:
+  - doi: 10.1016/0016-660X(76)90024-6
+    name: Infinite games and generalizations of first-countable spaces (Gruenhage)
+---
+
+Shown in the Theorem 3.7 of {{doi:10.1016/0016-660X(76)90024-6}}.
```
Comment (StevenClontz) [inline theorems/T000582.md:15]: I expect this is correct, but I think we need a few more details here.

**Review by StevenClontz — CHANGES_REQUESTED** [review]
See notes/suggestions above. Thanks for the contribution! Gary was my advisor; this paper is quite close to my heart. :-)

File: theorems/T000582.md (hunk 3 above)
Comment (Almanzoris) [reply theorems/T000582.md]: Alright, I have just added more details in its description.

**Comment by prabau** [comment]
@StevenClontz just curious, do you know why Gary chose the letter W in "W-space"?

**Comment by Almanzoris** [comment]
Hmm, I thought the W is because of the winning strategy under the perspective of player 1.

**Comment by StevenClontz** [comment]
> @StevenClontz just curious, do you know why Gary chose the letter W in "W-space"?

I'll spoil my upcoming paper with @jocelynbell - he once told me the "W" just stands for "Win" lol

**Comment by StevenClontz** [comment]
> Hmm, I thought the W is because of winning, under the perspective of player 1.

👆 😂

**Review by StevenClontz — APPROVED** [review]

**Comment by Almanzoris** [comment]
Jajaja, yeahh, it had to be that.

I hope your upcoming paper is successfully and satisfactorily published.

**Pull request merged**